include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -77,6 +77,7 @@ template <typename T>
 #ifdef ORT_API_MANUAL_INIT
 const OrtApi* Global<T>::api_{};
 inline void InitApi() { Global<void>::api_ = OrtGetApiBase()->GetApi(ORT_API_VERSION); }
+inline void InitApi(const OrtApi* api) { Global<void>::api_ = api; }
```

Comment:
Custom op dlls that define `ORT_API_MANUAL_INIT` need to call this overload of `InitApi` in order to be able to use C++ APIs that use `GetApi()` under the hood. Otherwise, `GetApi` will return null.